docs/articles/life-cycle/skipping-tests.rst

```diff
@@ -166,3 +166,37 @@ tests and are not available at this point.
                     tests/test_passkey.py::test_skip__lambda (ldap) SKIPPED (Server is not built with password policy support)
                     tests/test_passkey.py::test_skip__function (ldap) SKIPPED (Server is not built with password policy support)
                     tests/test_passkey.py::test_skip__function_and_reason (ldap) SKIPPED (Server is not built with password policy support)
+
+Preferred topology
+===================
+
+Sometimes, you want to maintain the flexibility to run tests against all topologies, but not always. Nightly builds and CI are examples where running a test against a single topology by default may make more sense than all topologies. This can reduce the resources required to run the tests and their execution times. All while using the same test code. This is possible by using the ``pytest.mark.preferred_topology`` marker. This marks the test with a default topology that will run, and for the comprehensive runs, "--mh-ignore-preferred-topology" will ignore this marker entirely.
+
+
+.. note::
+    If the CLI argument '--mh-topology' is used, the test will use that value and ignore the 'preferred_topology' marker.
```

Comment:
I think it would be better to provide more thorough description of the behavior it the main text instead of relying on a side note.

Reply:
It is also important to document the semantics - what happens if multiple markers are used and so on, what are the possible values and so on.